typedb/connection/cluster/server_client.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, address: str, credential: TypeDBCredential, parallelisation:
             self._channel_credentials = grpc.ssl_channel_credentials()
         self._channel, self._stub = self.new_channel_and_stub()
         self._databases = _TypeDBDatabaseManagerImpl(self.stub())
+        self._is_open = True
```

Comment:
we move the `is_open = True` down from the parent constructor into each sub-constructor